src/config.c

```diff
@@ -688,18 +718,18 @@ RSConfigOptions RSGlobalConfigOptions = {
          .getValue = getSearchThreads,
          .flags = RSCONFIGVAR_F_IMMUTABLE,
         },
-#ifdef POWER_TO_THE_WORKERS
+#ifdef MT_BUILD
         {.name = "WORKER_THREADS",
          .helpText = "Create at most this number of search threads",
          .setValue = setWorkThreads,
          .getValue = getWorkThreads,
          .flags = RSCONFIGVAR_F_IMMUTABLE,
         },
-        {.name = "ALWAYS_USE_THREADS",
+        {.name = "MT_MODE",
          .helpText = "Let ft.search and vector indexing be done in background threads as default if"
-                        "set to TRUE, use workers thread pool for operational needs only otherwise",
-         .setValue = setThreadsEnabled,
-         .getValue = getThreadsEnabled,
+                        "set to MT_MODE_RCP. MT_MODE_RCE use workers thread pool for operational needs only otherwise",
+         .setValue = setMtMode,
+         .getValue = getMtMode,
          .flags = RSCONFIGVAR_F_IMMUTABLE | RSCONFIGVAR_F_FLAG,
```

Comment:
If in not mistaken `RSCONFIGVAR_F_FLAG` means a boolean flag (which is not the case anymore...) , though I don't see where we use that flag... @GuyAv46 right?

Reply:
Yep. should be removed. And indeed nobody uses this flag anywhere today but...